repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -81,8 +81,8 @@ class KeystoneRequestHandler(identityServiceUri: String, akkaServiceClient: Akka
               case Failure(f) =>
                 Failure(IdentityCommunicationException("Token not found in identity response during admin authentication", f))
             }
-          case SC_REQUEST_ENTITY_TOO_LARGE | SC_TOO_MANY_REQUESTS =>
-            Failure(OverLimitException(buildRetryValue(serviceClientResponse), "Rate limited when getting admin token"))
+          case statusCode@(SC_REQUEST_ENTITY_TOO_LARGE | SC_TOO_MANY_REQUESTS) =>
```

Comment:
I was unaware of this syntax for binding a matching pattern to a variable. Cool. :+1: